docusaurus/docs/iOS/basics/integration.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Integration
+title: Installation
 ---
 
 # Integration
```

Comment:
- Would be nice to update the h1 (`#`) from `# Integration` to `# Installation`
- Would be nice to update the text `check our Integration page` on the pages `/chat/docs/sdk/ios/client/`, `/chat/docs/sdk/ios/uikit/` and `/chat/docs/sdk/ios/swiftui/` 
- Do we need to update the file name as well? If so, keep in mind that there are a lot of references across the docs and Readme.